src/footer/footer.macros.html

```diff
@@ -7,53 +7,42 @@
 {#
   * Renders a standard or simplified footer
 #}
-{% macro footer(simple=false, class=null, id=null, phone_number='888–498–0003') %}
+{% macro footer(data, simple=false, class=null, id=null) %}
 <footer class="footer{% if simple %} footer--simple{% endif %}{% if class %} {{ class }}{% endif %}"{% if id %} id="{{ id }}"{% endif %}>
   <div class="footer-top r-inner">
     {{ caller() }}
   </div>
   <div class="footer-bottom">
-    <nav class="footer-nav r-inner">
-      <div class="footer-nav-group footer-nav-group--bottom">
-        <span class="footer-nav-country-icons">
-          <a class="footer-nav-group-link footer-nav-group-link--country" href="?country_id=49">
-            {{ icon('usa', size='small') }}
+    <div class="footer-bottom-inner r-inner">
+
+      <span class="footer-bottom-item site-copyright">
+        &copy;
+        <script>document.write(new Date().getFullYear());</script>
+      </span>
+
+      <span class="site-legal-links footer-bottom-item">
+        <a class="" href="https://casper.com/privacy">{{ data.privacy }}</a>
+        <a class="" href="https://casper.com/terms">{{ data.terms }}</a>
+      </span>
+
+      <span class="site-address h-card footer-bottom-item">
+        <span class="p-street-address">{{ data.address }}</span>
+
+        {% if not simple %}
+        <span class="site-address-meta">
+          <a class="footer-bottom-item" href="tel:{{ data.phone_number|replace('-', '') }}">
```

Comment:
Agreed re positional names. I'd like to find a name that reflects the
content. `footer-nav` feels like a better class to replace `footer-top`.
What is now the bottom is not really a nav. It is site/company
details—copyright, address, contact, and links to terms and privacy.
Looking for a name to encompass the area that hold site/company
information.

On Mon, Jun 13, 2016 at 11:29 AM, Rob Wierzbowski <notifications@github.com>
wrote:

> In src/footer/footer.macros.html
> <https://github.com/CasperSleep/nightshade-core/pull/158#discussion_r66812988>
> :
>
> > +      <span class="footer-bottom-item site-copyright">
> > +        &copy;
> > +        <script>document.write(new Date().getFullYear());</script>
> > +      </span>
> > +
> > +      <span class="site-legal-links footer-bottom-item">
> > +        <a class="" href="https://casper.com/privacy">{{ data.privacy }}</a>
> > +        <a class="" href="https://casper.com/terms">{{ data.terms }}</a>
> > +      </span>
> > +
> > +      <span class="site-address h-card footer-bottom-item">
> > +        <span class="p-street-address">{{ data.address }}</span>
> > +
> > +        {% if not simple %}
> > +        <span class="site-address-meta">
> > +          <a class="footer-bottom-item" href="tel:{{ data.phone_number|replace('-', '') }}">
>
> Agree, no positional names in classes please.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CasperSleep/nightshade-core/pull/158/files/4a796ed164ca13b031a7ea3e0482ff3aaf03f0f9..e6ff78407f282366c5f519a0cac32d22b4fe8974#r66812988>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AADKZMa0C5o0sBcFFLU6hz0v2PAKst7aks5qLXdVgaJpZM4I0Pqz>
> .
>